packages/flutter/test/rendering/editable_gesture_test.dart

```diff
@@ -58,7 +57,12 @@ class _GestureBindingSpy extends AutomatedTestWidgetsFlutterBinding {
   PointerRouter get pointerRouter => _testPointerRouter;
 }
 
-class FakeEditableTextState extends TextSelectionDelegate with Mock { }
+class FakeEditableTextState extends TextSelectionDelegate {
```

Comment:
Could we abstract the noSuchMethod stuff out in our own mixin that we use in our tests to avoid the noSuchMethod ugliness everywhere?

Reply:
Actually, wasn't there a PR that adds that functionality to package:test?

Reply:
That is the Fake stuff, it has not rolled yet.

Reply:
Should we just wait for that?

Reply:
yeah, sgm